### PR TITLE
Send params to IGV view via POST request

### LINKED ISSUE #1653: Href passing to IGV too large - refactor jinja URL build
Passing href to IGV is a large chunk of nearly identical information. Is there a way to make Jinja accept a higher order structure?

Reference:
https://github.com/Clinical-Genomics/scout/pull/1548#discussion_r370552457

Example:
```
<a class="btn btn-outline-secondary btn-sm" href="/igv?sample=NA12878&amp;build=37&amp;bam=%2FUsers%2FMikael%2FWork%2Fgenomes%2FHG00096.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam&amp;bam=%2FUsers%2FMikael%2FWork%2Fgenomes%2FHG00096.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam&amp;bam=%2FUsers%2FMikael%2FWork%2Fgenomes%2FHG00096.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam&amp;bai=%2FUsers%2FMikael%2FWork%2Fgenomes%2FHG00096.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam.bai&amp;bai=%2FUsers%2FMikael%2FWork%2Fgenomes%2FHG00096.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam.bai&amp;bai=%2FUsers%2FMikael%2FWork%2Fgenomes%2FHG00096.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam.bai&amp;rhocall_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;rhocall_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;rhocall_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;rhocall_wig=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest.bw&amp;rhocall_wig=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest.bw&amp;rhocall_wig=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest.bw&amp;tiddit_coverage_wig=&amp;upd_regions_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;upd_regions_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;upd_regions_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;upd_sites_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;upd_sites_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;upd_sites_bed=%2FUsers%2FMikael%2FWork%2Fscout_dev2%2Fscout%2Fscout%2Fdemo%2Ftest_upd.sites.bed&amp;contig=7&amp;start=124491922&amp;stop=124492022&amp;center_guide=T" target="_blank">IGV viewer</a>
```

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #1653 , at least for the data already passed to the igv view in current master.

**How to test**:
1. Install the branch on scout stage (clinical-db)
1. Choose a recently updated case (for instance: https://scout-stage.scilifelab.se/cust002/F0000496)
1. Inspect a snv variant and make sure it looks as it should on igv
1. Inspect a sv variant and make sure it looks as it should on igv
1. Open the list of str variants and make sure the link to igv works and they are viewed correctly in this browser.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR (the branch should still be on clinical-db stage if nobody changes it)

### CODE REVIEW

**Comment by northwestwitch** [comment]
1. Test visualizing one **snv variant on chr 16**:

image

1. Test visualizing **one snv variant on MT chromosome**:

image


1. Test visualizing **one sv variant on chr 10, breakpoint 1**:

image

1. Same variant **breakpoint 2**:
image

1.  Same **sv variant, the whole variant**:
image

1. Test visualizing **one str variant**: FAAAAAAAIL!!!!
Needs fixing!!

**Comment by northwestwitch** [comment]
Fixed also the str variants page, and a the button for viewing **one str variant** in igv works as well:

image

**Comment by northwestwitch** [comment]
Ready for review!!

File: scout/server/blueprints/variant/templates/variant/sv-variant.html [hunk 1]
```diff
@@ -128,32 +128,67 @@
...
-                - Alignment:
-                <a class="btn btn-outline-secondary btn-sm" href="{{ url_for('alignviewers.igv', sample=case.sample_names, build=case.genome_build, bam=case.mt_bams, bai=case.mt_bais, contig=variant.chromosome, start=(variant.position - 500), stop=(variant.position + 500)) }}" target="_blank">IGV viewer</a>
-              {% elif case.bam_files %}
-                - Alignment:
-                <a class="btn btn-outline-secondary btn-sm" href="{{ url_for('alignviewers.igv', bam=case.bam_files, bai=case.bai_files, sample=case.sample_names, build=case.genome_build, contig=variant.chromosome, start=(variant.position - 500), stop=(variant.position + 500)) }}" target="_blank">IGV</a>
-              {% else %}
-                - BAM file(s) missing
-              {% endif %}
-              - <a class="btn btn-outline-secondary btn-sm" href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr{{ variant.chromosome }}:{{ variant.position }}&dgv=pack&knownGene=pack&omimGene=pack" target="_blank">UCSC</a>
+              <form role="form" name="bp1_form" action="{{ url_for('alignviewers.igv') }}" target="_blank" method="POST">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/sv-variant.html]: ⭐️ Sooo much more readable!

File: scout/server/blueprints/variant/templates/variant/sv-variant.html [hunk 2]
```diff
@@ -172,22 +207,38 @@
...
-              {% elif case.bam_files %}
-                - Complex rearrangement: check breakpoint alignments.
-              {% else %}
-                - BAM file(s) missing
-              {% endif %}
-              {% if variant.chromosome == variant.end_chrom %}
-              - <a class="btn btn-outline-secondary btn-sm" href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr{{ variant.chromosome }}:{{ variant.position }}-{{ variant.end }}&dgv=pack&knownGene=pack&omimGene=pack" target="_blank">UCSC</a>
-              {% endif %}
-            </div>
+              <form role="form" name="bp2_form" action="{{ url_for('alignviewers.igv') }}" target="_blank" method="POST">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/sv-variant.html]: Name? "alignment_length" perhaps?

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/sv-variant.html]: I'll fix it, thanks

**Comment by northwestwitch** [comment]
Merging!

**Pull request merged**